packages/shared-influx-pieces/src/index.ts

```diff
@@ -0,0 +1,43 @@
+// eslint-disable-next-line spaced-comment
+/// <reference lib="dom" />
+
+import type { BooleanString } from '@skyra/env-utilities';
+import { container } from '@skyra/http-framework';
+import { Client } from '@skyra/influx-utilities';
+
+let interactionCount: number;
+
+export function isInfluxInitialized() {
+	return Boolean(container.analytics);
+}
+
+export function initializeInflux(options: Client.Options = {}) {
+	if (!process.env.INFLUX_URL) return;
```

Comment:
Also check whether `INFLUX_ENABLED` is true (`envParseBoolean('INFLUX_ENABLED', true)`)